x/bank/keeper/msg_server.go

```diff
@@ -124,3 +124,30 @@ func (k msgServer) UpdateParams(goCtx context.Context, req *types.MsgUpdateParam
 
 	return &types.MsgUpdateParamsResponse{}, nil
 }
+
+func (k msgServer) SetSendEnabled(goCtx context.Context, msg *types.MsgSetSendEnabled) (*types.MsgSetSendEnabledResponse, error) {
+	if k.GetAuthority() != msg.Authority {
+		return nil, sdkerrors.Wrapf(govtypes.ErrInvalidSigner, "invalid authority; expected %s, got %s", k.GetAuthority(), msg.Authority)
+	}
+
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	if len(msg.SendEnabled) > 0 {
+		k.SetAllSendEnabled(ctx, msg.SendEnabled)
+	}
+	if len(msg.UseDefaultFor) > 0 {
+		k.DeleteSendEnabled(ctx, msg.UseDefaultFor...)
+	}
+	if msg.SetDefaultSendEnabled {
+		k.SetParams(ctx, types.NewParams(msg.DefaultSendEnabled))
+	}
```

Comment:
this is part of the above param changes, don't think we should duplicate it here as well

Reply:
I included it as part of this because I felt there might be a need to set both that default and some other send-enabled flags at the same time. That could be done with two governance proposals set to the same height, but that runs the risk of one passing and the other not in a situation when it needs to be both or neither.